### PR TITLE
ADD facebook login (PoC)

### PR DESCRIPTION
**Pull request by palmerabollo**
Add facebook login.
Logged users are saved in a new "users" collection.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -3,5 +3,10 @@
 module.exports = {
   "paypal": {
     "hosted_button_id": "ETEEHS7H7NBFJ"
+  },
+  "facebook": {
```
Comment (palmerabollo) [inline config.js]: These values correspond to a test app I have and are valid to try this new feature in localhost. You should go to http://developers.facebook.com/ and create a new app for Books4Equality.

**Comment by Blquinn** [comment]
I believe the current user login is sufficient, should I close this issue?

**Comment by SleekPanther** [comment]
Yeah. I think what the system we have is better than facebook because we can verify uvm email addresses.

**Comment by palmerabollo** [comment]
Sure. It was just a proof of concept.

**Pull request closed without merging**